portfolio/src/main/webapp/index.html

```diff
@@ -3,23 +3,22 @@
   <head>
     <meta charset="UTF-8">
     <title>My Portfolio</title>
-    <div id ="header">
-    <h1> Sean Tisdale Portfolio </h1>
-    <div id ="links">
-    <a href="https://github.com/SeanT18"> my personal github </a>
-    <br>
-    <a href="https://github.com/setisdale"> my school github </a>
-    <br>
-    <a href="www.linkedin.com/in/sean-tisdale"> mY Linkedin </a>
-    <br>
-    </div>
-    </div>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
+  <header>
```

Comment:
< header > tags should go in the < body >

https://stackoverflow.com/questions/9083816/html5-header-before-body